README.md

```diff
@@ -313,1001 +313,19 @@ Please note that this FAQ is provided for informational purposes only and is not
-* [`affinidi login create-group`](#affinidi-login-create-group)
-* [`affinidi login delete-config`](#affinidi-login-delete-config)
-* [`affinidi login delete-group`](#affinidi-login-delete-group)
-* [`affinidi login export-configs`](#affinidi-login-export-configs)
-* [`affinidi login export-groups`](#affinidi-login-export-groups)
-* [`affinidi login get-config`](#affinidi-login-get-config)
-* [`affinidi login get-group`](#affinidi-login-get-group)
-* [`affinidi login import-configs`](#affinidi-login-import-configs)
-* [`affinidi login import-groups`](#affinidi-login-import-groups)
-* [`affinidi login list-configs`](#affinidi-login-list-configs)
-* [`affinidi login list-groups`](#affinidi-login-list-groups)
-* [`affinidi login list-users-in-group`](#affinidi-login-list-users-in-group)
-* [`affinidi login remove-user-from-group`](#affinidi-login-remove-user-from-group)
-* [`affinidi login update-config`](#affinidi-login-update-config)
-* [`affinidi project create-project`](#affinidi-project-create-project)
-* [`affinidi project get-active-project`](#affinidi-project-get-active-project)
-* [`affinidi project list-projects`](#affinidi-project-list-projects)
-* [`affinidi project select-project`](#affinidi-project-select-project)
-* [`affinidi project update-project`](#affinidi-project-update-project)
-* [`affinidi search`](#affinidi-search)
-* [`affinidi start`](#affinidi-start)
-* [`affinidi stop`](#affinidi-stop)
-* [`affinidi token create-token`](#affinidi-token-create-token)
-* [`affinidi token delete-token`](#affinidi-token-delete-token)
-* [`affinidi token get-token`](#affinidi-token-get-token)
-* [`affinidi token list-tokens`](#affinidi-token-list-tokens)
-* [`affinidi token update-token`](#affinidi-token-update-token)
-* [`affinidi whoami`](#affinidi-whoami)
-
-## `affinidi autocomplete [SHELL]`
-
-display autocomplete installation instructions
+# Command Topics
+
+* [`affinidi autocomplete`](docs/autocomplete.md) - display autocomplete installation instructions
+* [`affinidi commands`](docs/commands.md) - list all the commands
+* [`affinidi generate`](docs/generate.md) - Use these commands to generate code to get started or to scaffold your solution
+* [`affinidi help`](docs/help.md) - Display help for affinidi.
+* [`affinidi iam`](docs/iam.md) - Use these commands to manage policies for access configuration
+* [`affinidi login`](docs/login.md) - Use these commands for user login configuration and group management
+* [`affinidi project`](docs/project.md) - Use these commands to manage your projects
+* [`affinidi search`](docs/search.md) - Search for a command.
+* [`affinidi start`](docs/start.md) - Log in to Affinidi
+* [`affinidi stop`](docs/stop.md) - Log out from Affinidi
+* [`affinidi token`](docs/token.md) - Use these commands for Personal Access Token (PAT) management
+* [`affinidi whoami`](docs/whoami.md) - Returns user's subject, projects, and token details from the current session.
```

Comment:
Lets move these to the `### Available commands` section above.

Reply:
Moved the command topics to the `### Available commands` section and updated the text.